packages/config/src/library/validation.ts

```diff
@@ -2,6 +2,7 @@ import { utils } from "ethers";
 import { ZodIssueCode, RefinementCtx } from "zod";
 
 export const STORE_SELECTOR_MAX_LENGTH = 16;
```

Comment:
since this is no longer a selector, should we rename this to `STORE_RESOURCE_NAME_MAX_LENGTH` or something?

Reply:
could do!

Reply:
done